src/components/delete-with-confirmation-icon.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Button, IconButton } from '@mui/material';
+import React, { useState } from 'react';
+import { useTranslator } from '../hooks';
+import CloseIcon from '@mui/icons-material/Close';
+
+export function DeleteWithConfirmationIcon(props: {
```

Comment:
```suggestion
export function ConfirmDeleteButton(props: {
```

Reply:
As a part of PR #152 `DeleteWithConfirmationButton` was added (button with modal confirmation popup). Taking this into account, does  renaming that element into `ConfirmDeleteButton` and this into `ConfirmDeleteIcon` makes sense?